src/services/services.ts

```diff
@@ -3840,26 +3841,23 @@ namespace ts {
 
             return { isMemberCompletion, isNewIdentifierLocation, entries };
 
-            function getJavaScriptCompletionEntries(): CompletionEntry[] {
+            function getJavaScriptCompletionEntries(sourceFile: SourceFile, uniqueNames: Map<any>): CompletionEntry[] {

```

Comment:
because this effectively should be Set<string>. Currently it is initially populated as `string -> Symbol` and then as `string -> string` and thus is typed as `string -> any`. I'll check if I can switch both places to use `string -> string`.